clouddriver-docker/clouddriver-docker.gradle

```diff
@@ -9,6 +9,7 @@ dependencies {
 
   implementation "org.springframework.boot:spring-boot-actuator"
   implementation "org.springframework.boot:spring-boot-starter-web"
+  implementation "org.springframework.cloud:spring-cloud-context"
```

Comment:
Is this dependency actually used for something?

Reply:
we are using it for `@RefreshScope` annotation in
https://github.com/spinnaker/clouddriver/pull/5593/files#diff-392529703c92986db659767f07fbb4eca5e5d4c1f2603af40c5e0608d7f4bb91R56